test/unit/middlewares/application-validator.test.ts

```diff
@@ -40,10 +40,10 @@ describe("application validator test", function () {
     });
 
     it("should not call the next function if any of the values which have a wordCount restriction doesn't contain the expected number of words", async function () {
-       const rawData = {
-         ...applicationsData[6],
-         whyRds: 'jfaskdfjsd'
-       };
+      const rawData = {
+        ...applicationsData[6],
+        whyRds: "jfaskdfjsd",
+      };
```

Comment:
Why this changed?

Reply:
They were done by lint in my vscode system

Reply:
so remove this changes